lib/llm/providers/ollama.rb

```diff
@@ -0,0 +1,55 @@
+# frozen_string_literal: true
+
+module LLM
+  ##
+  # The Ollama class implements a provider for
+  # [Ollama](https://ollama.ai/)
+  class Ollama < Provider
+    require_relative "ollama/error_handler"
+    require_relative "ollama/response_parser"
+
+    HOST = "localhost"
+    DEFAULT_PARAMS = {model: "llama3.2", stream: false}.freeze
+
+    ##
+    # @param secret (see LLM::Provider#initialize)
+    def initialize(secret, **)
+      super(secret, HOST, 11434, ssl: false, **)
```

Comment:
This looks weird to me. Why are we using 3 positional arguments, and one keyword argument ? The interface is this:
```
LLM.open(secret, options)
```
Everything that goes into `options` should be a keyword argument.